Airports/EGOE/Runway.txt

```diff
@@ -1,3 +1,3 @@
-04  22  043	223 N052.52.14.230 W002.32.09.680 N052.52.32.520 W002.31.42.100
-10  28  100	280 N052.52.24.370 W002.32.22.540 N052.52.20.820 W002.31.45.040
-17  35  169	349 N052.52.37.930 W002.31.50.760 N052.52.12.600 W002.31.42.320
\ No newline at end of file
+04  22  042 222 N052.52.14.230 W002.32.09.680 N052.52.14.230 W002.32.09.680
```

Comment:
```suggestion
04  22  042 222 N052.52.14.230 W002.32.09.680 N052.52.32.520 W002.31.42.100
```
Not sure why this was changed, but you have both the 04 and 22 THR co-ords in the same place, I think it was correct as was?